src/main/java/com/commerce/backendserver/auth/infra/jwt/JwtProvider.java

```diff
@@ -74,10 +80,4 @@ public void validateToken(String token) {
             throw CommerceException.of(AuthError.TOKEN_INVALID);
```

Comment:
```suggestion
            throw CommerceException.of(TOKEN_INVALID);
```

기존에 관습적으로 사용하시던 대로, static import 사용하는 것도 좋아보여요.